nfsv4/session.c

```diff
@@ -176,7 +176,9 @@ status reclaim_complete(nfs4 c)
     // if true, current fh is the object for which reclaim is complete
     push_be32(r->b, 0);
     boolean bs;
-    print_buffer("reclaim", r->b);
+    if (config_boolean("NFS_RECLAIM_TRACE", false)) {
```

Comment:
Is there an ongoing need for this parameter? I wonder whether we could have a few debugging levels rather than many flags.